README.md

```diff
@@ -0,0 +1,42 @@
+# External Secrets
+
+The External Secrets Kubernetes operator reads information from a third party service
+like [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/)  and automatically injects the values as [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/).
+
+Multiple people and organizations are joining efforts to create a single External Secrets solution based on existing projects. If you are curious about the origins of this project, check out this issue [kubernetes-external-secrets/issues/47](https://github.com/external-secrets/kubernetes-external-secrets/issues/47) and this [PR](https://github.com/external-secrets/kubernetes-external-secrets/pull/477) and the discussion that triggered this.
+
+
+<a name="features"></a>
+
+# Planned Features
+
+- Support to multiple Provider stores (AWS Secret Manager, GCP Secret Manger, Vault and more) simultaneously.
+- Multiple External Secrets operator instances for different contexts/environments.
+- A custom refresh interval to sync the data from the Providers to keep your Kubernetes Secrets up to dater.
```

Comment:
I think it would be even better:
> A custom refresh interval to sync the data from the Providers, syncing your Kubernetes Secrets up to date.

Also, there's a type on the last word `dater -> date`.